src/ncp-spinel/SpinelNCPInstance-DataPump.cpp

```diff
@@ -381,6 +381,16 @@ SpinelNCPInstance::driver_to_ncp_pump()
 				syslog(LOG_INFO, "[->NCP] CMD_RESET tid:%d", SPINEL_HEADER_GET_TID(mOutboundBuffer[0]));
 			} else if (mOutboundBuffer[1] == SPINEL_CMD_NET_CLEAR) {
 				syslog(LOG_INFO, "[->NCP] CMD_NET_CLEAR tid:%d", SPINEL_HEADER_GET_TID(mOutboundBuffer[0]));
+			} else if (mOutboundBuffer[1] == SPINEL_CMD_PEEK) {
+				uint32_t address;
+				uint16_t count;
+				spinel_datatype_unpack(mOutboundBuffer, mOutboundBufferLen, "CiLS", NULL, NULL, &address, &count);
```

Comment:
You should either be setting default values for `address` and `count`, or be checking the return value of `spinel_datatype_unpack()` for errors. Otherwise, in the event of a parser failure, you will end up logging garbage values.